packages/react-icons/package.json

```diff
@@ -52,7 +52,7 @@
   },
   "peerDependencies": {
     "@griffel/react": "^1.0.0",
-    "react": ">=16.8.0 <=18.0.0"
+    "react": ">=16.8.0 <19.0.0"
```

Comment:
Allows React 18, the same as in `@fluentui/react-components`, https://github.com/microsoft/fluentui/blob/8fcefaeacac2e2cc2cd7ca5b76c1ca5e7e547682/packages/react-components/react-components/package.json#L80.